src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -300,15 +307,25 @@ export class NetCoreDebugHelper implements DebugHelper {
                 const installProgress = (content: string) => {
                     output.appendLine(content);
                 };
-                const dockerExecOptions : DockerExecOptions = { interactive: true, progress: installProgress };
+                const dockerExecOptions: DockerExecOptions = { interactive: true, progress: installProgress };
 
                 await dockerClient.exec(containerName, installDebugger, dockerExecOptions);
             },
             'Debugger installed',
             'Unable to install the .NET Core debugger.'
         );
 
-        return `${debuggerPath}/vsdbg`;
+        return `${installPath}/vsdbg`;
+    }
+
+    private async isDebuggerInstalled(containerName: string, debuggerPath: string): Promise<boolean> {
+        const dockerClient = new CliDockerClient(new ChildProcessProvider());
+        const osProvider = new LocalOSProvider();
+        const command: string = osProvider.os === 'Windows' ?

```

Comment:
So the only difference on a Windows host is to use double quotes instead of single? There's no way to that one quoting technique works on both?

Reply:
Yes that is right. Single quote isn't working on windows.